app/konnect/org-management/sso.md

```diff
@@ -61,7 +58,38 @@ You can configure custom IdP-specific behaviors in the **Advanced Settings** of
     - `name`: Used as the {{site.konnect_short_name}} account's `full_name`.
     - `email`: Used as the {{site.konnect_short_name}} account's `email`.
     - `groups`: Used to map users to teams defined in the team mappings upon login.
+{% endnavtab %}
+{% navtab SAML %}
+1. Log in to {{site.konnect_saas}}, click {% konnect_icon organizations %} **Organization**, and then select **Auth Settings**.
+
+2. Click **Configure provider** under **SAML**. 
+
+3. Enter the **Metadata URL** from your IdP in the **Identity Provider Metadata URL** field, or paste the **Metadata XML** in to the Identity **Provider Metadata XML** field.
+
+4. In the **Organization Login Path** field, enter the unique string that matches the one in Okta. For example: `examplepath`.
+
+   {{site.konnect_short_name}} uses this string to generate a custom login
+   URL for your organization.
+
+   Requirements:
+    * The path must be unique across all {{site.konnect_short_name}} organizations.
+      If your desired path is already taken, you must choose another one.
+    * The path can be any alphanumeric string.
+    * The path does not require a slash (`/`).
+
+5. Click **Save**.
```

Comment:
After clicking 'Save', we should add additional instructions to guide user to go back to their SAML IdP to configure the `SP Entity ID` and `Login URL` on the third-party system.